### ISSUE TITLE
Wazuh KIbana App 3.2.0 cannot be used in kibana 6.1.2 altough i'm using Wazuh Kibana App 3.2.0 for Kibana 6.1.2

### ISSUE THREAD

**Issue by nizarakbarm** [wazuh/wazuh-kibana-app#265]
I'm having problem when using wazuh kibana app 3.2.0-6.1.2 in kibana 6.1.2. It's showing an error about API Mismatch and expect wazuh 3.1.0.
![1519030064_1547_19022018_1184x616](https://user-images.githubusercontent.com/12729998/36369186-e3bae2d4-158c-11e8-84f6-64043c55aca3.png)

**Comment by jesusgn90** [comment]
Hi @nizarakbarm the error you are facing could be caused by different things. When our App throws that error is because the API version stored on Elasticsearch is different from the version you are receiving when requesting the Wazuh API. We are going to make some checks before, please ensure the following mandatory settings:

```
# cat /usr/share/kibana/plugins/wazuh/package.json | grep version
```
The expected output should be:

```
  "version": "3.2.0", 
  "version": "6.1.2"
```
The above output confirms us that you are using in fact the package [wazuhapp-3.2.0_6.1.2.zip](https://packages.wazuh.com/wazuhapp/wazuhapp-3.2.0_6.1.2.zip). 

Now we are going to check the Wazuh API version:

```
# curl -s <wazuh_api_ip>:<wazuh_api_port>/version -u <wazuh_api_user>:<wazuh_api_pass>

// Eg: curl -s localhost:55000/version -u foo:bar
```
If you've installed Wazuh 3.2.0 you should see the next output for the above command:

```
{"error":0,"data":"v3.2.0"}
```

Now we are going to check the Elasticsearch installed version as follow:

```
# curl <elastic_search_ip>:9200 -s | grep number
// Eg: curl localhost:9200 -s | grep number
```
You should see the next output:

```
   "number" : "6.1.2"
```

If all of the above commands are matching with your installation, it should be fine, but please check also the next request to Elasticsearch:

```
# curl <elasticsearch_ip>:9200/.wazuh-version/_search?pretty -s | grep app-version
// Eg: curl localhost:9200/.wazuh-version/_search?pretty -s | grep app-version
```
You should see the next output:

```
    "app-version" : "3.2.0",
```

Let us know if you are mistmatching one or more of the citated commands so we can continue helping you. Have a nice day.

Best regards,
Jesús

**Comment by mmccarn** [comment]
This sequence of commands seems to solve the "API version mismatch" error:
```
systemctl stop kibana
curl -XDELETE localhost:9200/.kibana
curl -XDELETE localhost:9200/.wazuh
curl -XDELETE localhost:9200/.wazuh-version
rm -rf /usr/share/kibana/optimize/bundles
systemctl start kibana
# wait 5 minutes for bundles to finish rebuilding; check status using
# systemctl status kibana
```

**Warning**: these commands delete all custom settings including the wazuh api username, password, url, and port. 

You can check your current API settings here:
http://<wazuh-server-name-or-ip:5601/app/wazuh#/settings/?tab=api

**Comment by jesusgn90** [comment]
Hi @mmccarn thanks for collaborate with us on the issues and tell us about your results, that's great. On the other hand, @nizarakbarm , how is your environment now? Is it fine just now? Also I want to tell you that we have released Wazuh 3.2.1 and a new Wazuh App compatible with Elastic 6.2.2, have a nice day.

Best regards,
Jesús

**Comment by nizarakbarm** [comment]
I am using wazuh 3.2.1 and elastic 6.2.2. Thanks for your response
